src/_getMetaData.ts

```diff
@@ -0,0 +1,17 @@
+import { AMetaData, ContentRecord } from "@activeviam/activeui-sdk-5.0";
+
+/**
+ * Returns the metadata of the file with id `fileId` under `structure`.
+ */
+export function _getMetaData<MetaData extends AMetaData = AMetaData>(
+  structure: ContentRecord,
+  pathToParentFolder: string[],
+  fileId: string,
+): MetaData {
+  const metadataRecord = [
+    ...pathToParentFolder,
+    fileId,
+    `${fileId}_metadata`,
+  ].reduce((acc, id) => acc.children![id], structure);
```

Comment:
I feel like we're using the `!` with confidence, I'm not sure if we should worry about the case where a bug happened somewhere above, and `pathToParentFolder` is inconsistent with `fileId`. Should we throw maybe? The error could be cleanly caught and added to the error report in the end. What do you think? Maybe I'm overthinking this

Reply:
I don't mind the idea, although I would tend to throw higher up if there is a structural issue: at this stage you expect the `pathToParentFolder` to have been computed by a first pass within the structure anyway.
I would rather not complexify the PR for now to unblock the rest.

Reply:
Okay, I'm fine with not doing this here!